CSS/styles-about.css

```diff
@@ -0,0 +1,489 @@
+  margin-bottom: 53px;
+}
+
+.lemorse-mssg {
+  padding: 40px 40px 10px 40px;
+  font: 400 15px 'Lato', sans-serif;
+  color: rgb(51, 51, 51);
+  text-align: center;
+  line-height: 22px;
+  width: 90%;
+}
+
+.cntc-email {
+  font: 700 14px 'Lato', sans-serif;
+  color: rgb(51, 51, 51);
+  text-decoration: underline;
+}
+
+.underline {
+  width: 10%;
+  height: 1px;
+  background-color: #ec5242;
+  margin: 8px auto;
+}
+
+.moreline {
+  width: 50%;
+}
+
+.memorable-logo {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 10px;
+  border: 1px solid #e4e4e4;
+  padding: 51px 25px 0 25px;
+}
+
+.memorable-logo-tt {
+  text-align: center;
+  font-size: 24px;
+  font-family: 'Lato', sans-serif;
+}
+
+.competition-logo {
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+  margin: 0;
+  font-size: 16px;
+  line-height: 24px;
+  color: #333;
+}
+
+.big-logo {
+  width: 75%;
+  height: 20%;
+  margin: 55px;
+  border: 1px solid #e4e4e4;
+}
+
+.previous-Researches {
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  display: flex;
+  margin: 20px;
+  gap: 40px;
+}
+
+.previous-Researches h2 {
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+}
+
+.previous-Researches p {
+  font-family: 'Lato', sans-serif;
+  text-align: center;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+.research-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 30px;
+  gap: 15px;
+  width: 100%;
+}
+
+.research-date {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-size: 50px;
+  font-weight: 700;
+}
+
+.research-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 5px;
+  background-image: url(../images/research-img-R1.jpg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  box-shadow: inset 0 0 0 100px rgb(239 81 60 / 67%);
+  width: 80%;
+  height: 200px;
+  color: #fff;
+}
+
+.research-2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 5px;
+  background-image: url(../images/research-img-R2.png);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  box-shadow: inset 0 0 0 100px rgb(239 81 60 / 67%);
+  width: 80%;
+  height: 200px;
+  color: #fff;
+}
+
+.research-1:hover {
+  cursor: pointer;
+  box-shadow: none;
+}
+
+.research-2:hover {
+  cursor: pointer;
+  box-shadow: none;
+}
+
+.sponsors {
+  background-color: #2e2e2e;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 5px;
+  padding: 20px 0 30px 0;
+}
+
+.spon-t {
+  text-align: center;
+  font-size: 24px;
+  font-family: 'Lato', sans-serif;
+  font-weight: 500;
+  padding: 3px;
+  color: #fff;
+}
+
+.supports-logo {
+  display: flex;
+  justify-content: center;
+  gap: 25px;
+  width: 90%;
+}
+
+.sf-icons {
+  filter: grayscale(95%);
+  cursor: pointer;
+  width: 50px;
+  height: 50px;
+}
+
+footer {
```

Comment:
- The footer has dark background color on the about page. Sticking to the provided template is a project requirement.

Reply:
Thanks for the requested change I also made sure that it looks more like the original template!😄